ROMFS/px4fmu_common/init.d/airframes/50000_generic_ground_vehicle

```diff
@@ -12,6 +12,7 @@
 #
 # @maintainer
 #
+# @board px4_fmu-v2 exclude
```

Comment:
This one was intentionally kept for the sake of px4_fmu-v2_rover (yes we need a better solution).

Reply:
Added back.